src/cascadia/Remoting/Peasant.cpp

```diff
@@ -126,14 +127,17 @@ namespace winrt::Microsoft::Terminal::Remoting::implementation
     // - <none>
     // Return Value:
     // - <none>
-    void Peasant::Summon()
+    void Peasant::Summon(const Remoting::SummonWindowBehavior& summonBehavior)
     {
-        _SummonRequestedHandlers(*this, nullptr);
+        auto localCopy = winrt::make_self<implementation::SummonWindowBehavior>(summonBehavior);

```

Comment:
Why are we copying it? Is it because it's an OOP object, and we can get a local?

Reply:
Yea basically. Stash the local copy so that if during the `SummonRequestedHandlers`, the `Monarch` dies, then the Peasant can still complete the summon, without the handler knowing that the args are OOP.